### DISCUSSION TITLE
Stars added into list should not be visible outside the list

### DISCUSSION THREAD

**Discussion by SakethKanchi** [community/community#8676] [Other Feature Feedback, Questions, & Ideas] [answered]
Stars added into list should not be visible outside the list. Having 20-50 stars and arranging them into lists is god-sent but if the stars are still shown outside the list then it's a bit confusing and more messy

**Comment by brianlovin** [comment, marked as answer]
Thanks for the feedback @SakethKanchi! Several people have mentioned something similar, so I'll take this back to the team to discuss how we should think about this view.

**Reply by Kalipiczy** [reply]
![touch-to-start](https://user-images.githubusercontent.com/99761023/157338824-57f5058f-8e6f-4789-a74d-20b560b744b0.png)

**Comment by jasonkuhrt** [comment]
I have 1.5k stars. I want to organize them but that's no easy task if I can't know which ones have been put into lists already vs not. So I really need a way to filter stars by those not in any list.
